src/Catalog/CatalogWriterBase.cs

```diff
@@ -70,7 +70,7 @@ public Task Commit( IGraph commitMetadata, CancellationToken cancellationToken)
 
             if (_batch.Count == 0)
             {
-                return;
+                return null;
```

Comment:
Fixed

---
In reply to: [57127064](https://github.com/NuGet/NuGet.Services.Metadata/pull/60#discussion_r57127064) [](ancestors = 57127064)